typescript/lib/constructor_calls.ts

```diff
@@ -31,12 +36,184 @@ export function newGetFileGenerationOptions(filesToGenerateSet: Set<string>): Ma
 }
 
 export function newGetLoadStaticFilesArgs(serviceId: ServiceID, staticFilesToCopyStringSet: Map<String, boolean>): LoadStaticFilesArgs {
-    const result = new LoadStaticFilesArgs();
+    const result: LoadStaticFilesArgs = new LoadStaticFilesArgs();
     result.setServiceId(String(serviceId));
     const staticFilesMap = result.getStaticFilesMap();
     for (let staticFildID in staticFilesToCopyStringSet) {
         staticFilesMap.set(staticFildID, staticFilesToCopyStringSet[staticFildID]);
     }
 
+    return result;
+} 
+
+// // ====================================================================================================
+// //                                    Network Context
+// // ====================================================================================================
+
+export function newGetLoadLambdaArgs(lambdaId: LambdaID, image: string, serializedParams: string): LoadLambdaArgs {
```

Comment:
It looks like all these functions have `newGet...` prefixed when they should just be `new....`